### PR TITLE
A0-2272: Fix copying the address

### PR DESCRIPTION
**Pull request by youPickItUp**

### CODE REVIEW

File: packages/extension-ui/src/components/Address.spec.tsx [hunk 1]
```diff
@@ -248,12 +248,17 @@ const genesisHashTestSuite = (account: AccountTestGenesisJson, withAccountsInCon
...
-    //   expect(wrapper.find('CopyToClipboard').at(0).prop('text')).toEqual(expectedEncodedAddress);
-    //   expect(wrapper.find('CopyToClipboard').at(1).prop('text')).toEqual(expectedEncodedAddress);
-    // });
+    it('Copy address to clipboard on click and do not go to account menu', () => {
+      // eslint-disable-next-line deprecation/deprecation
+      (document.execCommand as jest.Mock).mockClear();
+      const stopPropagation = jest.fn();
+
+      wrapper.find('.fullAddress').simulate('click', { stopPropagation });
+      // the first CopyToClipboard is from the identicon, the second from the copy button
```
Comment (jalooc) [inline packages/extension-ui/src/components/Address.spec.tsx:257]: Isn't that comment a remaining?

File: packages/extension-ui/src/components/Address.spec.tsx [hunk 2]
```diff
@@ -248,12 +248,17 @@ const genesisHashTestSuite = (account: AccountTestGenesisJson, withAccountsInCon
...
+      // eslint-disable-next-line deprecation/deprecation
+      expect(document.execCommand).toHaveBeenCalledWith('copy');
+      expect(stopPropagation).toHaveBeenCalled();
```
Comment (jalooc) [inline packages/extension-ui/src/components/Address.spec.tsx:260]: That's an internal detail - do we really care about it in tests? If we're after "not go to account menu", would it be possible to instead check if some router (?) function was **not** invoked?

File: packages/extension-ui/src/components/Address.tsx [hunk 3]
```diff
@@ -199,59 +194,62 @@ function Address({
...
+          <div className='addressDisplay'>
+            <CopyToClipboard
+              onCopy={onCopy}
+              text={address || ''}
+            >
               <div
                 className='fullAddress'
                 data-field='address'
-                onClick={_onCopy}
+                onClick={onCopyClickStopPropagation}
```
Comment (jalooc) [inline packages/extension-ui/src/components/Address.tsx:246]: Why is stopPropagation needed here?

**Review by jalooc — APPROVED** [review]

File: packages/extension-ui/src/components/Address.tsx (hunk 3 above)
Comment (youPickItUp) [reply packages/extension-ui/src/components/Address.tsx:246]: Because higher in DOM hierarchy we have an onClick that navigates to an account menu

**Pull request merged**